mix.exs

```diff
@@ -1,36 +1,46 @@
 defmodule EctoStateMachine.Mixfile do
   use Mix.Project
 
+  @project_url "https://github.com/asiniy/ecto_state_machine"
+  @version "0.0.4"
+
   def project do
     [
       app: :ecto_state_machine,
-      version: "0.0.4",
+      version: @version,
       elixir: "~> 1.2",
       elixirc_paths: elixirc_paths(Mix.env),
       build_embedded: Mix.env == :prod,
       start_permanent: Mix.env == :prod,
       deps: deps,
-
+      source_url: @project_url,
+      homepage_url: @project_url,
       description: "State machine pattern for Ecto. I tried to make it similar as possible to ruby's gem 'aasm'",
       package: package
    ]
   end
 
-  defp elixirc_paths(:prod), do: ["lib"]
-  defp elixirc_paths(:dev),  do: ["lib", "test/dummy/web", "test/dummy/lib"]
-  defp elixirc_paths(:test), do: ["lib", "test"]
+  defp elixirc_paths(:test), do: elixirc_paths ++ ["test/support", "test/dummy"]
```

Comment:
I explained the essential of tabularization in another PR. If

> ["test/support", "test/dummy"] will be ["test/support"] in next PRs

let's move this kind of change to the next PR